includes/defaults.inc.php

```diff
@@ -742,6 +743,7 @@
 $config['discovery_modules']['stp']            = 1;
 $config['discovery_modules']['cisco-otv']      = 1;
 $config['discovery_modules']['ntp']            = 1;
+$config['discovery_modules']['loadbalancers']  = 1;
```

Comment:
We now have the ability to enable / disable these on a per os basis so set them to 0 here and enable them in the os. See `$os = 'linux';` in definitions.inc.php to see.